lib/data/demos.dart

```diff
@@ -49,22 +50,56 @@ import 'package:gallery/themes/material_demo_theme_data.dart';
 
 const _docsBaseUrl = 'https://api.flutter.dev/flutter';
 
+enum GalleryDemoCategory {
+  study,
+  material,
+  cupertino,
+  other,
+}
+
+extension GalleryDemoExtension on GalleryDemoCategory {
+  String get name => describeEnum(this);
+
+  String displayTitle(GalleryLocalizations localizations) {
+    switch (this) {
+      case GalleryDemoCategory.material:
+        return 'MATERIAL';
+      case GalleryDemoCategory.cupertino:
+        return 'CUPERTINO';
+      case GalleryDemoCategory.other:
+        return localizations.homeCategoryReference;
+      case GalleryDemoCategory.study:
```

Comment:
Seems a bit odd for this case to do nothing, does Dart make you list it?

Reply:
Yes, you have to list all enum members in switch clauses